Documentation~/CinemachineBindingModes.md

```diff
@@ -0,0 +1,94 @@
+# Binding Modes
+
+The binding mode is the coordinate space Unity uses to interpret the Camera offset from the target and the damping.
```

Comment:
Not _is_.  The binding mode _defines_ the coordinate space

Reply:
Done.